src/main/java/frc/robot/commands/IntakeDefaultCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.filter.Debouncer.DebounceType;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDefaultCommand extends CommandBase{
```

Comment:
There seems to be a `CrocodileDefaultCommand` and an `IntakeDefaultCommand` so a bit confused about which one is the real one that should be used. They seem to be identical so I will let my comments on `CrocodileDefualtCommand` apply to this file as well.

Reply:
deleting this one